.circleci/config/jobs/@jobs.yml

```diff
@@ -237,23 +229,18 @@ integration_tests_proxy:
 eks_integration_tests:
   machine:
     image: ubuntu-2004:202010-01
-    enabled: true
     docker_layer_caching: true
   working_directory: ~/kubernetes-monitor
   steps:
     - checkout
     - install_python_requests
+    - setup_node16
     - run:
         name: Create temp dir for logs
         command: mkdir -p /tmp/logs/test/integration/eks
     - run:
         name: Integration tests EKS
-        # WARNING! Do not use the step "setup_node16" here - the call to "nvm use 16" breaks the tests!
```

Comment:
Just wondering how did you pass this step now, as I can see that you have been using the ```setup_node16``` step now 🙂   And why was it breaking the tests before?

Reply:
@minsiyang so I just decided to try it with `nvm use` and I don't have clue what is was breaking :) Tried searching in commit history. So just trying to figure :)

Reply:
From what I remember it didn't interact well with python